owscheck.py

```diff
@@ -195,3 +196,8 @@ def get_service(self):
 
     def get_layer_names(self):
         return self._layer_names
+
+    def _reduced_bbox(self, bbox):
+        return [b*0.1 for f in bbox]
```

Comment:
Sure it works ?
eg there is no data outside 48.8, 1.34, 51.08, 4.28...

Reply:
```
>>> bbox = [ 48.8, 1.34, 51.08, 4.28 ]
>>> [b*0.1 for b in bbox]
[4.88, 0.134, 5.108, 0.428]
```
Resulting bbox is out of layer definition

Reply:
If we are to take 10% of the `[left, bottom, right, top]` bbox, then the computation should rather be 
`[left + 0.95*(right-left), bottom + 0.95*(top-bottom), right - 0.95*(right-left), top - 0.95*(top-bottom) ]`

Reply:
And really, I'm wondering if 10% is still not too big. Why not just 1% ? 
As long as we have a `200 OK` at the end ..